tests/subworkflows/nf-core/annotation_snpeff/main.nf

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { ANNOTATION_SNPEFF } from '../../../../subworkflows/nf-core/annotation_snpeff/main'
+
+workflow annotation_snpeff {
+    input = [
+        [ id:'test' ], // meta map
+        file(params.test_data['sarscov2']['illumina']['test_vcf'], checkIfExists: true)
+    ]
+
+    ANNOTATION_SNPEFF ( input, "WBcel235.99", [] )
```

Comment:
All parameter test?

Reply:
yes, no need for more, we can add more in args, this is the minimal

Reply:
Ooh I see what you mean, then no I don't want to test for that.
This empty parameter is for already downloaded cache supplied by the user.
It's a edge case